modules/core/src/webgl/buffer.js

```diff
@@ -2,23 +2,55 @@ import GL from '@luma.gl/constants';
 import Resource from './resource';
 import Accessor from './accessor';
 import {assertWebGL2Context} from '../webgl-utils';
-import {getGLTypeFromTypedArray, getTypedArrayFromGLType} from '../webgl-utils/typed-array-utils';
-import {log} from '../utils';
-import assert from '../utils/assert';
+import {getGLTypeFromTypedArray, getTypedArrayFromGLType} from '../webgl-utils';
+import {log, assert, checkProps} from '../utils';
 
 const DEBUG_DATA_LENGTH = 10;
 
+// Shared prop checks for constructor and setProps
+const DEPRECATED_PROPS = {
```

Comment:
So we are getting rid of the accessor?

Reply:
On the contrary, I am enforcing the accessor in favor of allowing accessor fields directly in the Buffer props. 

What these rules are saying is that:

`new Buffer(gl, {size: ...})` should be replaced with `new Buffer(gl, {accessor {size: ...}})`